sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -392,15 +399,26 @@ public bool IsRunning
             Argument.AssertNotNullOrEmpty(consumerGroup, nameof(consumerGroup));
             Argument.AssertNotNullOrEmpty(connectionString, nameof(connectionString));
 
-            clientOptions = clientOptions?.Clone() ?? new EventProcessorClientOptions();
+            clientOptions = clientOptions ?? new EventProcessorClientOptions();
+
+            ProcessingConsumerOptions = new EventHubConsumerClientOptions
+            {
+                RetryOptions = clientOptions.RetryOptions
```

Comment:
This is why we'd clone.   These could change out from under us, which we don't want.

Reply:
Actually, that was my bad. I intended to call `clientOptions.RetryOptions.Clone()`.

Reply:
I would need to be careful with the `ConnectionOptions` as well, in fact. I agree that it might be easier to introduce bugs this way, so I'll add the `Clone` call back.